src/applications/ivc-champva/10-10D/pages/ApplicantRelOriginPage.jsx

```diff
@@ -66,7 +66,7 @@ export function ApplicantRelOriginPage(props) {
   };
   return ApplicantRelationshipPage(newProps);
 }
-export function ApplicationRelOriginReviewPage(props) {
+export function ApplicantRelOriginReviewPage(props) {
```

Comment:
Fixing a previous typo in the name of this component - also adjusted wherever it is imported, and internal to the component itself.